src/main/scala/uk/gov/homeoffice/drt/ports/config/Lba.scala

```diff
@@ -22,7 +24,13 @@ object Lba extends AirportConfigLike {
       Queues.NonEeaDesk -> 45
     ),
     defaultWalkTimeMillis = Map(T1 -> 600000L),
-    terminalPaxSplits = List(T1).map(t => (t, defaultPaxSplits)).toMap,
+    terminalPaxSplits = Map(T1 -> SplitRatios(
```

Comment:
For understanding , how do we get split ration different to default ?

Reply:
This was my fault as I told you we could use the defaults. But LBA doesn't have egates, and the defaults include a split to the egates. So this override just makes all machine readable and all B5JSSK pax go to the desks rather than splitting between desks and egates